test/browser/components.test.js

```diff
@@ -5,15 +5,15 @@ import {
 	teardown,
 	getMixedArray,
 	mixedArrayHTML,
-	serializeHtml
+	serializeHtml,
+	sortAttributes,
+	spyAll
 } from '../_util/helpers';
 import { div, span, p } from '../_util/dom';
 
 /** @jsx createElement */
 const h = createElement;
 
-let spyAll = obj => Object.keys(obj).forEach(key => sinon.spy(obj, key));
```

Comment:
Removed this copy of the `spyAll` function and replaced it with the same function declared in `helpers.js`. Same for `sortAttributes` below